aptos-move/framework/aptos-framework/sources/account.spec.move

```diff
@@ -598,41 +647,33 @@ spec aptos_framework::account {
     /// The system reserved addresses is @0x1 / @0x2 / @0x3 / @0x4 / @0x5  / @0x6 / @0x7 / @0x8 / @0x9 / @0xa.
     spec create_framework_reserved_account(addr: address): (signer, SignerCapability) {
         aborts_if spec_is_framework_address(addr);
-        include CreateAccountAbortsIf {addr};
+        include CreateAccountAbortsIf { addr };
         ensures signer::address_of(result_1) == addr;
         ensures result_2 == SignerCapability { account: addr };
     }
 
-    spec fun spec_is_framework_address(addr: address): bool{
-        addr != @0x1 &&
-        addr != @0x2 &&
-        addr != @0x3 &&
-        addr != @0x4 &&
-        addr != @0x5 &&
-        addr != @0x6 &&
-        addr != @0x7 &&
-        addr != @0x8 &&
-        addr != @0x9 &&
-        addr != @0xa
+    spec fun spec_is_framework_address(addr: address): bool {
+        addr != @0x1 && addr != @0x2 && addr != @0x3 && addr != @0x4 && addr != @0x5 && addr
```

Comment:
Not sure whether we should format this.

Reply:
In general, it seems like having a way for the user to specify parts of code that should be left as-is without formatting is likely very important, because a user might have spent a lot of time hand-formatting certain parts of code to read a particular way.

There are two ways to handle this:
1. Comments above and below the block of code that should not be formatted (`// move-format off` and `// move-format on`). This is what is done in a lot of C++ tools. The positives are that it could be applied to an arbitrary portion of the code (eg., for three functions out of several, or a subset of statements within a function). The negative is that it is not really first class to the language.
2. Have an attribute like `#[move-format::skip]` on blocks of code that the user wants formatting to be skipped. This is what rustfmt does, however, it has the drawback of requiring lots of annotations or workarounds (see https://stackoverflow.com/questions/67288537/how-can-i-switch-off-rustfmt-for-a-region-of-code-instead-of-a-single-item), and also, we would need to support these attributes (but probably only for compiler v2).

Reply:
@robinlzw , since option 2 requires support of the attribute, we may want to go with option 1